templates/modulesTakenPriorToOthers.html

```diff
@@ -0,0 +1,100 @@
+$def with (modulePairs, allAySems, studentCounts, studentPriorCount, moduleA, moduleB, targetAySem, studentsInModuleB)
+
+$var title:View Modules Taken Prior to Others
+
+$if modulePairs is not None:
+	<div class="container-fluid">
+		<h1 class="text-center"><b>Modules that are taken prior to other modules</b></h1>
+		<p class="text-center">Shows all pairs of modules where module A is taken prior to module B</b></p>
+		<br>
+		<div class="modules-taken-prior-search-form">
+			<form action="/moduleTakenPriorToOthers" method="POST">
```

Comment:
Validation is needed for both module code inputs.

You want to leave it to UI department to fix or you want to implement it?